hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/PipelinePlacementPolicy.java

```diff
@@ -99,9 +99,8 @@ boolean meetCriteria(DatanodeDetails datanodeDetails, int nodesRequired) {
       try {
         pipeline = stateManager.getPipeline(pid);
       } catch (PipelineNotFoundException e) {
-        LOG.error("Pipeline not found in pipeline state manager during" +
-            " pipeline creation. PipelineID: " + pid +
-            " exception: " + e.getMessage());
+        LOG.debug("Pipeline not found in pipeline state manager during" +
+            " pipeline creation. PipelineID: {}" + pid + e);
```

Comment:
This line still is not correct. The way SLF4J works, is it substitutes the {} for each parameter in turn. If the last parameter is a Throwable, it prints the stack trace after the log message. So I think it should be:
```
LOG.debug("Pipeline not found in pipeline state manager during" +
            " pipeline creation. PipelineID: {}", pid, e);
```

Reply:
Updated.